roles/server/defaults/main.yml

```diff
@@ -2,8 +2,8 @@
 cache_name: benchmark
 # Location of the cache configuration to use for the named cache
 cache_file: files/cache.xml
-# Optional way to override the server image used by the benchmar
-server_image: quay.io/infinispan/server
+# Optional way to override the server image used by the benchmark
+server_image: "quay.io/infinispan/server:15.0"
```

Comment:
I don't know if we want to change this default or not... User should probably override instead if they want a specific version.